virtool/postgres.py

```diff
@@ -63,5 +63,10 @@ def to_dict(self):
 
     return row
 
+@classmethod
+def to_list(cls):
+        return [e.value for e in cls.__members__.values()]
+
```

Comment:
Indentation problem at line 68.

The  classmethod decorator is better used in this case like so:
```py
Enum.to_list = classmethod(func)
```

Also, you should not modifiy `Enum` as it is imported from another package in the standard library. Subclass enum or find a better solution in the `enum` package documentation. Do a general Google search on a solution for this before writing a subclass of `Enum`.